src/components/Notification/index.js

```diff
@@ -0,0 +1,83 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { Notification as NotificationItem } from '@santiment-network/ui'
+import SlideEntry from './Animated/SlideEntry'
+import styles from './Notification.module.scss'
+
+class Notification extends Component {
```

Comment:
The name is semantically incorrect. It implies the existence of only **one** notification. A more appropriate name would be `NotificationStack`